astro/marble/pkg-plist

```diff
@@ -233,7 +231,7 @@ lib/marble/plugins/PntPlugin.so
 lib/marble/plugins/PositionMarker.so
 lib/marble/plugins/PostalCode.so
 lib/marble/plugins/ProgressFloatItem.so
-%%GPS%%lib/marble/plugins/QtPositioningPositionProviderPlugin.so
+%%GPS_OR_WEBENGINE%%lib/marble/plugins/QtPositioningPositionProviderPlugin.so
```

Comment:
This looks bizarre. The options framework wouldn't sub this. The only extra requirement for this plugin is qt6-positioning. I think positioning should just be added as a hard dependency so this can be an unconditional file. I think this ended up this way because the GPS and WEBENGINE options add qt6-location which depends on qt6-positioning. In qt5, positioning is part of the location port.